chrome/js/modules/request.js

```diff
@@ -2018,6 +2018,10 @@ pm.request = {
 
         var macKey = $('#request-helper-basicAuth-username').val();
         var macSecret = $('#request-helper-basicAuth-password').val();
+
+        macKey = pm.envManager.convertString(macKey);
```

Comment:
These changes were previously (prior to my PRs) implemented in the requester.js file - which is a derived file - but were not implemented in this file, which is one of the source files are that used to generate the requester.js file.